packages/audio-block/src/settings.ts

```diff
@@ -0,0 +1,41 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    AssetChooserObjectType,
+    AssetInputSize,
+    FileExtension,
+    IconEnum,
+    defineSettings,
+} from '@frontify/guideline-blocks-settings';
+
+import { TextPosition } from './types';
+
+export const AUDIO_ID = 'audio';
+export const AUDIO_EXTENSIONS = [FileExtension.Flac, FileExtension.Mp3, FileExtension.Ogg, FileExtension.Wav];
+export const ATTACHMENTS_SETTING_ID = 'attachments';
+
+export const settings = defineSettings({
+    basics: [
+        {
+            id: AUDIO_ID,
+            type: 'assetInput',
+            label: 'Audio',
+            info: 'Select an audio file to play.',
+            size: AssetInputSize.Small,
+            extensions: AUDIO_EXTENSIONS,
```

Comment:
```suggestion
            extensions: FileExtensionSets.Audio,
```

Reply:
Removed the AUDIO_EXTENSIONS within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/79566eea36ab5407175e44a3271bfaf87ecbb73e)